projects/plugins/jetpack/tests/e2e/lib/pages/wp-admin/block-editor.js

```diff
@@ -18,7 +18,9 @@ export default class BlockEditorPage extends WpPage {
 	}
 
 	get searchBlockFldSel() {
-		return '.block-editor-inserter__search-input';
+		// There are 2 classes here because the class changed in Gutenberg 11.2 but is not yet in the WP bundled version.
+		//todo to remove .block-editor-inserter__search-input once WP will include GB version 11.2
```

Comment:
Some IDEs highlight TODO comments when it matches below pattern.

```suggestion
		//TODO: to remove .block-editor-inserter__search-input once WP will include GB version 11.2
```

Reply:
Yeah, that's the reason I marked the comment as such. I'm currently using WebStorm for which simple `todo` is enough to match, I just assumed most editors use the same pattern.